### ISSUE TITLE
UniDict support

### ISSUE THREAD

**Issue by Golrith** [mekanica/mekanica#101]
#### Issue description:
Switching over to Mekanica causes UniDict to no longer standardise the recipes in all the various machines. I'm using Thermal Foundation as my common standard in all machines/mods.

UniDict works by mod ID name, but doesn't seem able to detect Mekanica correctly (despite still being internally still called Mekanism)

This is totally low priority, as I can always make a load of crafttweaker scripts to force the overrides.

**Comment by pupnewfster** [comment]
The only idea I have about this is at some point I moved registering ore dict for Mekanica items to the proper spot from being rather late. It could be that it actually is properly seeing the mekanism items as ore dict'd so getting things differently. One thing though that may help if you are manually standardizing, Mekanica allows you to use OreDict bracket handlers/compound ingredients for removing recipes, so it should be relatively easy to remove all types of a specific type that is duplicated.

**Comment by pupnewfster** [comment]
Slight update on this. @dizzyd was able to reproduce this even though I wasn't. I believe it is likely that Mekanism actually also has this issue in the more recent updates, but we will try to get a fix out soon as I have some ideas of how to fix it. Mainly just waiting on dizzy to get a chance to test it.

**Comment by Golrith** [comment]
Must admit I haven't noticed under the latest version of Mek, but then I wasn't actually looking.

Decided for myself to stop being lazy and remove UniDict and script it myself.

Keep up the good work!
